book/graphics.md

```diff
@@ -70,45 +63,45 @@ while True:
 ```
 
 ::: {.print-only}
-![A flowchart of event handling cycle](im/graphics-cycle.png)
+![Figure 1: Flowchart of an event handling cycle.](im/graphics-cycle.png)
 :::
 
 Here, `pendingEvent` first asks the desktop environment for recent
 mouse clicks or key presses, then `handleEvent` calls your application
 to update state, and then `drawScreen` redraws the window. This *event
-loop*\index{event loop} pattern is common in many applications, from
+loop* \index{event loop} pattern (see Figure 1) is common in many applications, from
```

Comment:
I'm a little unsure why this space got gobbled up (it's a common LaTeX problem) but adding a space here should be safe.

Reply:
Confirmed that it looks fine on web and PDF.